src/shared/components/palette/case-flag/read-case-flag-field.component.ts

```diff
@@ -2,8 +2,8 @@ import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { CaseTab } from '../../../domain';
 import { FieldsUtils } from '../../../services/fields';
-import { PaletteContext } from '../base-field/palette-context.enum';
 import { AbstractFieldReadComponent } from '../base-field/abstract-field-read.component';
+import { PaletteContext } from '../base-field/palette-context.enum';
```

Comment:
@johnbenjamin-hmcts - is `PaletteContext` being used anywhere or is this a left-over import?